Harvest.Web/ClientApp/src/Tickets/TicketsContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { Project, Ticket } from "../types";
+import { ProjectHeader } from "../Shared/ProjectHeader";
+import { TicketTable } from "./TicketTable";
+
+interface RouteParams {
+  projectId?: string;
+}
+
+export const TicketsContainer = () => {
+  const { projectId } = useParams<RouteParams>();
+  const [project, setProject] = useState<Project>();
+  const [tickets, setTickets] = useState<Ticket[]>([]);
+
+  useEffect(() => {
+    const cb = async () => {
+      const response = await fetch(`/Project/Get/${projectId}`);
+
+      if (response.ok) {
+        const proj: Project = await response.json();
+        setProject(proj);
+      }
+    };
+
+    cb();
+  }, [projectId]);
+
+  useEffect(() => {
+    const cb = async () => {
+      const response = await fetch(`/Ticket/GetList?projectId=${projectId}`);
+
+      if (response.ok) {
+        setTickets(await response.json());
+      }
+    };
+
+    cb();
+  }, [projectId]);
+
+  if (project === undefined) {
```

Comment:
not sure the best way to handle this, but you'll probably want to check that both tickets and projects have loaded.  As it is now, once the project loads then loading will go away and it could show 0 tickets even though we are still waiting for the ticket API call to come back.  Perhaps you'll want to not initialize to `[]` and check for undefined on the tickets array too?